qa/multi-cluster-search/src/test/java/org/elasticsearch/search/CCSDuelIT.java

```diff
@@ -580,13 +581,14 @@ public void testSortByField() throws Exception {
 
     public void testSortByFieldOneClusterHasNoResults() throws Exception {
         assumeMultiClusterSetup();
-        // set to a value greater than the number of shards to avoid differences due to the skipping of shards
+        // setting aggs to avoid differences due to the skipping of shards when matching none
```

Comment:
this is interesting, the previous suggested we were skipping can match, that would have been a better work-around, but if your code change was required, that means that we were not skipping can match? was it outdated then?

Reply:
The previous comment was outdated, there was a refactor of this method a while back and the `setPreFilterShardSize` the comments referees to was removed